plugins/gpio/README.md

```diff
@@ -0,0 +1,22 @@
+# GPIO
+
+## Introduction
+
+This plugin sets GPIO outputs either high or low before and/or after an
+update has been deployed.
+
+## Quirk Use
+
+This plugin uses the following plugin-specific quirks:
+
+### GpioPrepare
+
+The GPIO bits to set during the update prepare phase, e.g. `5:high,6:low`.
+
+Since: 1.7.6
+
+### GpioCleanup
+
+The GPIO bits to set during the update cleanup phase, e.g. `5:high,6:low`.
```

Comment:
I think this is most likely too simplistic, no?  What about edge vs level triggers?  What about debounce?  Active high vs active low?

Reply:
This is really for *output* signals only; I don't think we want to get into the game of listening for GPIO events.